topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1244 @@
+---
+layout: tutorial_hands_on
+topic_name: transcriptomics
+tutorial_name: scrna
+---
+
+<style>
+blockquote img {
+  width: 50%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+.tutorial table { 
+  width:unset;
+  margin-left:auto;
+  margin-right:auto;
+}
+
+.tutorial table thead th {
+  border-bottom: 2px solid #777; 
+  text-align: left;
+}
+
+.tutorial table td:first-child, .tutorial table th:first-child  {
+  border-right: 2px solid #777; 
+  font-weight: bold;
+  text-align: left;
+}
+
+.tutorial table td:not(first-child), .tutorial table th:not(first-child) {
+  text-align: center;
+  border-right: 1px solid #ddd;
+  border-left: 1px solid #ddd;
+}
+
+</style>
+
+
+# Introduction
+{:.no_toc}
+
+The advent of single-cell RNA sequencing has provided the means to explore samples at the individual cell level, enabling a greater understanding of the development and function of such samples by the characteristics of their constituent cells. These cells can be classified into different cell subpopulations or "clusters" by the variation of gene expression exhibited by individual cells that share a common profile -- i.e. cells in the same cluster exhibit similar profiles of differential expression in the same set of related genes.
+
+By identifying significant genes in each cluster, cell types can be inferred and similarity between clusters can be determined based on their proximity to one another. If the cells all come from the same sample tissue, the cell types will likely correspond to the different stages of cell differentiation expected of that tissue, wherein a lineage/heirarchy could potentially be derived.
+
+
+ ![Facets of Cellular Identity](../../images/scrna_nbt3711.png "The various facets of cellular identity")
+
+This tutorial is in part inspired by aspects of the [Hemberg workflow](https://hemberg-lab.github.io/scRNA.seq.course/) at the Sangar insititute, as well as the [CGATOxford](https://github.com/CGATOxford/UMI-tools) workflow. The barcoding follows the CelSeq2 protocol and uses the lane configuration as that utilized by the Freiburg MPI Grün lab.
```

Comment:
Give the information about barcoding  where you explain barcoding - not here, because here nobody knows was barcoding/barcodes is/are